src/main/java/seedu/address/logic/commands/AddClientCommand.java

```diff
@@ -22,6 +22,8 @@ public abstract class AddClientCommand extends Command {
     public static final String CLIENT_PARAMETERS = PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL";
+    public static final String CLIENT_RESTRICTIONS = "Restrictions: "
```

Comment:
why do we skip the email and name restrictions though? if we want to specify restrictions, should we specify all of them?